bootstrap/google_compute_k8s_ubuntu_16.04_master.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+set -e
+cd ~
+
+# ------------------------------------------------------------------------------------------------------------------------
+# These values are injected into the script. We are explicitly not using a templating language to inject the values
+# as to encourage the user to limit their use of templating logic in these files. By design all injected values should
+# be able to be set at runtime, and the shell script real work. If you need conditional logic, write it in bash
+# or make another shell script.
+#
+#
+TOKEN="INJECTEDTOKEN"
+PORT="INJECTEDPORT"
+#
+#
+# ------------------------------------------------------------------------------------------------------------------------
+
+curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
+touch /etc/apt/sources.list.d/kubernetes.list
+sh -c 'echo "deb http://apt.kubernetes.io/ kubernetes-xenial main" > /etc/apt/sources.list.d/kubernetes.list'
+
+apt-get update -y
+apt-get install -y \
+    socat \
```

Comment:
Since you're not using socat/ebtables directly, I assume you're just installing them because they're dependencies of kubelet. I would just stick to listing toplevel packages you're directly depending on, since apt-get automatically installs transitive dependencies for you.